internal/providers/sns.provider.go

```
@@ -2,20 +2,21 @@ package providers
 
 import (
 	config "auth-plus-notification/config"
+	"context"
 	"errors"
 
 	"github.com/aws/aws-sdk-go/aws"
```

golangci-lint (2.12.2)

[error] SA1019: "github.com/aws/aws-sdk-go/aws" is deprecated: aws-sdk-go is deprecated. Use aws-sdk-go-v2. See https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-aws-sdk-for-go-v1-on-july-31-2025/. 

(staticcheck)